### DISCUSSION TITLE
Desktop sharing

### DISCUSSION THREAD

**Discussion by Lusciniamegarhynchos** [BearWare/TeamTalk5#2122] [Q&A]
I want to improve the cursor tracking of the desktop access control, is there a setting to speed it up?
We would like to improve cursor tracking when remote control is done while sharing the same screen with Mr. A and Mr. B.
Please advice.

**Comment by bear101** [comment]
There's not a configuration option to speed it up.

**Reply by Lusciniamegarhynchos** [reply]
Thank you for your reply!
I am very disappointed.
Since it is so highly functional, I think users will be even more pleased if the latency in this area is about the same as that of  Windows shadow session.

**Comment by computertech1012** [comment]
Hi,
I would also like to see such a feature. It would be good to remote control directly from TeamTalk, rather than having to install 3rd party applications like TeamViewer or VNC. This would also be good for being able to remote other devices for presenting, or to have the ability to remote control a server for troubleshooting via TeamTalk. This would also be helpful for controling another pc from anywhere because the username can be added to the desktop share.
Thanks for reading!
